packages/datagateway-dataview/src/page/pageContainer.component.test.tsx

```diff
@@ -445,4 +446,29 @@ describe('PageContainer - Tests', () => {
       wrapper.find(PageContainer).children().first().state('modifiedLocation')
     ).toEqual(newLocation);
   });
+
+  it('should return the homepage if current path equals homepage route', () => {
+    const homepageLocation = createLocation(paths.homepage);
```

Comment:
I like the idea of how you made use of `paths` here as values can be kept in one place and changes will not need to be made in multiple place if the value of `homepage` needs to be changed in future. I wonder if it's worth doing the same thing for the other tests of this file too?

Reply:
That's what I thought. Worth a refactor, I'd say

Reply:
This has also been done